os/os.go

```diff
@@ -0,0 +1,172 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package os provides gNOI os operations.
+package os
+
+import (
+	"context"
+	"fmt"
+	"io"
+
+	log "github.com/golang/glog"
+	ospb "github.com/openconfig/gnoi/os"
+	"github.com/openconfig/gnoigo/internal"
+)
+
+// InstallOperation represents the parameters of a Install operation.
+type InstallOperation struct {
+	transferReq *ospb.TransferRequest
```

Comment:
Rename this to req for consistency with the other operations?

Reply:
done